content/version/2/0/code_of_conduct.md

```diff
@@ -11,26 +11,32 @@ We as members, contributors, and leaders pledge to make participation in our com
 
 We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
 
+We believe that a diverse community where people treat each other with respect is stronger, more vibrant and has more potential contributors and more sources for ideas. We aim for more diversity.
+
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our community include:
 
 * Demonstrating empathy and kindness toward other people
+* Demonstrating mutual respect and tolerance towards other people
 * Being respectful of differing opinions, viewpoints, and experiences
 * Giving and gracefully accepting constructive feedback
 * Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
 * Focusing on what is best not just for us as individuals, but for the overall community
+* Using inclusive, gender-neutral language in the documents we provide and when we talk to people. When referring to a group of people, we aim to use gender-neutral terms like “team”, “folks”, “everyone”.
 
 Examples of unacceptable behavior include:
 
 * The use of sexualized language or imagery, and sexual attention or
   advances of any kind
 * Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
+* Harassment
```

Comment:
this seems less strong to me; without the qualifier, people may assume private harassment is not covered.

Reply:
Ahh I thought the complete opposite that it made it more generic. 
The intention behind this change was to make it more generic: Harassment of any kind in any sphere is not allowed.

Reply:
I think both are identical (since everything would be public or private), it’s just about whether the default understanding would include both. Given people’s tendency to assume that a code of conduct doesn’t get a say on what happens in private, i think the explicit indication might be better.

Reply:
Reverted this change.